R/dates_secuTrial.R

```diff
@@ -27,58 +27,19 @@
 dates_secuTrial <- function(x, ...) UseMethod("dates_secuTrial", x)
 datetimes_secuTrial <- function(x, ...) UseMethod("datetimes_secuTrial", x)
 
+
+
+
 #' @rdname dates_secuTrial
 #' @export
 dates_secuTrial.secuTrialdata <- function(object, ...) {
   if (object$export_options$dated) warning("dates already added")
-
   table_names <- object$export_options$data_names
   names(table_names) <- NULL
-  # not for meta tables
-  table_names <- table_names[!table_names %in% object$export_options$meta_names]
   # get language and internationalization dictionary for items table
   dict <- object$export_options$dict_items
+  obs <- lapply(table_names, function(obj) .convert_dates(obj, object, dict, ...))
```

Comment:
can this be simplified to 
```
obs <- lapply(table_names, .convert_dates, object = object, dict = dict, ...)
```
?

Reply:
you're right. i'll implement the change. thanks!